apptools/scripting/recorder.py

```diff
@@ -404,10 +403,7 @@ def save(self, file):
         """Save the recorded lines to the given file.  It does not close
         the file.
         """
-        if six.PY3:
-            file.write(self.get_code())
-        else:
-            file.write(six.text_type(self.get_code(), encoding="utf-8"))
```

Comment:
This change is not a flake8 cleaning, it is just related to dropping pyhton 2 support #120 
I added it because the later case below with `six.PY2` had a flake8 error, so I removed it, and it felt appropriate to remove them together.

Reply:
This sparked my opening of #230 which makes the same change, but goes further to replace all uses of `six` throughout the code base.